pattern-library/html/boilerplate-blank-RTL.html

```diff
@@ -0,0 +1,54 @@
+<!--
+edX Pattern Library: Boilerplate HTML Template: Header + Footer (RTL layout direction!)
+
+NOTES:
+
+* base template with details on how to handle primary/secondary assets, metadata, and basic view-level UI (on canvas vs. conditional and off-canvas elements)
+* right-to-left (RTL) layout configuration have been added
+* see https://github.com/edx/ux-pattern-library/wiki/Styleguide:-HTML#right-to-left-rtl-support for more information
+-->
+
+<!DOCTYPE html>
+<html class="no-js" lang="en" dir="rtl">
+    <head dir="rtl">
+        <meta charset="utf-8">
+        <!-- always force latest IE rendering engine or request chrome rrame -->
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
```

Comment:
Given that the Pattern Library has no control over the request, I think it makes sense to include this. I'll add a comment mentioning that this might not work for all browsers, but it should work for most. For example, this well regarded HTML5 boilerplate library includes this:

https://html5boilerplate.com/

In particular:

https://github.com/h5bp/html5-boilerplate/blob/5.3.0/dist/doc/html.md#indexhtml

Does that sound like a reasonable approach?